openpype/hosts/maya/plugins/create/create_review.py

```diff
@@ -29,6 +38,43 @@ class CreateReview(plugin.MayaCreator):
     useMayaTimeline = True
     panZoom = False
 
+    # Overriding "create" method to prefill values from settings.
+    def create(self, subset_name, instance_data, pre_create_data):
+
+        members = list()
+        if pre_create_data.get("use_selection"):
+            members = cmds.ls(selection=True)
+
+        project_name = get_current_project_name()
+        asset_doc = get_asset_by_name(project_name, instance_data["asset"])
+        task_name = get_current_task_name()
+        preset = lib.get_capture_preset(
+            task_name,
+            asset_doc["data"]["tasks"][task_name]["type"],
+            subset_name,
+            get_project_settings(project_name),
+            self.log
+        )
+        self.log.debug(
+            "Using preset: {}".format(
+                json.dumps(preset, indent=4, sort_keys=True)
+            )
+        )
+
+        with lib.undo_chunk():
+            instance_node = cmds.sets(members, name=subset_name)
+            instance_data["instance_node"] = instance_node
+            instance = CreatedInstance(
+                self.family,
+                subset_name,
+                instance_data,
+                self)
+            self._add_instance_to_context(instance)
+
+            self.imprint_instance_node(instance_node,
+                                       data=instance.data_to_store())
+            return instance
```

Comment:
This should be done by a call to `super(CreateReview, self).create` right instead of reimplementing it?

Reply:
Doing some attribute modifications so cant call super.

Reply:
Where is the custom attribute modification? :) And what type of attribute modification is being done that couldn't be done prior to the super call to the input arguments?

Reply:
Should be there now :)